src/messages/MMonMgrReport.h

```diff
@@ -48,6 +49,23 @@ class MMonMgrReport : public PaxosServiceMessage {
     paxos_encode();
     encode(health_checks, payload);
     encode(service_map_bl, payload);
+
+    if (!HAVE_FEATURE(features, SERVER_NAUTILUS) ||
```

Comment:
only SERVER_MIMIC here

Reply:
@liewegas Ah, of course! (facepalm) Fixed and repushed.